lib/_components/appCard/macro.njk

```diff
@@ -1,3 +1,20 @@
+{% from "appBadges/macro.njk" import appBadges %}
+
 {% macro appCard(params) %}
   {%- include "./template.njk" -%}
+{% endmacro %}
+
+{# Card specific macros #}
```

Comment:
Embedding template code in macros steps away from the 'standard' of including elsewhere.
Though I like the approach to scoping embedded macros like this.
Did you consider importing other templates, with different filenames. e.g.

```njk
{% macro cardHeader(item) %}
  {%- include "./template_cardHeader.njk" -%}
{% endmacro %}
{% macro cardInfo(item) %}
  {%- include "./template_cardInfo.njk" -%}
{% endmacro %}
```